frontend/packages/dev-console/src/components/topology/topology-utils.ts

```diff
@@ -35,9 +36,14 @@ export const getServiceBindingStatus = ({ FLAGS }: RootState): boolean =>
 export const getCheURL = (consoleLinks: K8sResourceKind[]) =>
   _.get(_.find(consoleLinks, ['metadata.name', 'che']), 'spec.href', '');
 
-export const getEditURL = (gitURL: string, gitBranch: string, cheURL: string) => {
+export const getEditURL = (vcsURI?: string, gitBranch?: string, cheURL?: string) => {
+  if (!vcsURI) {
+    return null;
+  }
```

Comment:
@rottencandy Why did this become needed in the z-stream but not in the 4.7 version?

Reply:
This change is in 4.7 as well, with #7216

Reply:
Ah, gotcha! Misunderstood the description when I took a quick look at this.